src/engine/controls/macrocontrol.h

```diff
@@ -14,15 +14,29 @@ class MacroControl : public EngineControl {
     MacroControl(QString group, UserSettingsPointer pConfig, int number);
 
     void trackLoaded(TrackPointer pNewTrack) override;
+    void process(const double dRate, const double dCurrentSample, const int iBufferSize) override;
+
+    bool isRunning() const;
+
+    enum Status {
+        Off = 0,
+        Armed = 1,
+        Recording = 2,
+        Recorded = 3,
+        Running = 4,
```

Comment:
I think "Running" doesn't really fit. Playing? Enabled?

Reply:
"Enabled" is a really bad idea, as it already has a meaning in this context ;)
"Playing" sounds good